File: js/08.js
const numRandomico = Math.floor(Math.random() * 10)  + 1
const numUser = 4
let tentativas = 0

while(tentativas <= 5) {
    tentativas += 1
    if(numRandomico != numUser) {
        console.log('errou, tente novamente')
    }

    else {
        console.log('acertou')
        break
    }
}

File: ruby/08.rb
while true
  num_randomico = rand(1..10)

  puts 'Digite um numero para ver se acerta'
  puts 'tente adinhar o numero do computador'
  numUser = gets.chomp.to_i

  if numUser == num_randomico
    puts 'parabens voce acertou'
    break

  else
    puts "Você errou! O número correto era #{num_randomico}. Tente novamente."
  end
end


